src/components/profile/add-place/add-place.js

```diff
@@ -119,7 +119,8 @@ export default class AddPlace extends Component {
               value={this.state.filesValue}
               multiple
               required
-              onChange={this.fileSelectedHandler} />
+              onChange={this.fileSelectedHandler}
+              accept=".png, .jpeg, .jpg" />
```

Comment:
I did it on server previous sprint

![image](https://user-images.githubusercontent.com/48828135/70325174-16ecaa00-183a-11ea-8fd1-3b075bdf1417.png)

Reply:
In my opinion having a server-only validation is a bad idea, everything needs to be validated on the client side too

Reply:
We don't need extra server requests

Reply:
user will not see any another files except theese. I think this is good, coz 2 factor validation can't be bad)